app/javascript/App/bundles/common/components/Banner.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react'
+import { withStyles, createStyles } from '@material-ui/core/styles'
+import { Button, Paper, Typography, Grid } from '@material-ui/core'
+
+const styles = (theme) =>
+  createStyles({
+    mainFeaturedPost: {
+      backgroundColor: '#253640',
+      color: '#d7d7d7',
+      marginBottom: theme.spacing.unit * 4,
+      boxShadow: 'none',
+    },
+    mainFeaturedPostContent: {
+      padding: `${theme.spacing.unit * 4}px`,
+      [theme.breakpoints.up('sm')]: {
+        paddingLeft: 0,
+        paddingRight: 0,
+      },
+    },
+    header: {
+      marginBottom: '0.5em !important',
+      fontSize: '1.5rem',
+      textAlign: 'center',
+    },
+    subheader: {
+      fontSize: '0.8rem',
+      textAlign: 'center',
+      [theme.breakpoints.up('sm')]: {
+        fontSize: '1rem',
+      },
+    },
+    cta: {
+      backgroundColor: '#23adf0',
+      color: '#fff',
+      borderRadius: '2px',
+      '&:hover': {
+        backgroundColor: '#23adf0',
+        boxShadow: '0 2px 10px rgba(14, 151, 255, 0.4)',
+      },
+    },
+  })
+
+const Banner = ({ classes }) => (
+  <Paper className={classes.mainFeaturedPost} square={true}>
+    <Grid
+      container={true}
+      direction="column"
+      justify="center"
+      alignItems="center"
+      className={classes.mainFeaturedPostContent}
+    >
+      <Grid item={true} md={true}>
+        <Typography className={classes.header} color="inherit">
+          Financial intelligence for cryptocurrency
+        </Typography>
+      </Grid>
+      <Grid item={true} md={true}>
+        <Typography
+          className={classes.subheader}
+          color="inherit"
+          paragraph={true}
+        >
+          Uncover buy and sell opportunities through data science backed
+          blockchain analytics.
+        </Typography>
+      </Grid>
+      <Grid item={true} md={true}>
+        <Button
```

Comment:
@jlei523 We probably want to instrument Heap yeah? Can you help provide @anthotsang with the appropriate instrumentation docs and data attribute for this?

Reply:
@anthotsang Here's the documentation: https://app.nuclino.com/CoinFi/Product/Instrumenting-Heap-Analytics-Events-via-data-attributes-e800f8d2-414b-47cd-bccd-20d29c1438ce

For this event, let's name it "homepage-click-signals-cta"

Reply:
@jlei523 Thanks. I've added the attribute. Based on the above documentation, I assume I don't need to add the event on Heap, but please let me know if I should do that too.